Commands/ContentTypes/RemoveContentType.cs

```diff
@@ -23,7 +23,7 @@ public class RemoveContentType : SPOWebCmdlet
         [Parameter(Mandatory = true, Position=0, ValueFromPipeline=true, HelpMessage="The name or ID of the content type to remove")]
         public ContentTypePipeBind Identity;
 
-        [Parameter(Mandatory = false)]
+        [Parameter(Mandatory = false, HelpMessage = "Overwrites the output file if it exists.")]
```

Comment:
Incorrect help message. Specifying the Force parameter will skip the confirmation question.